app/build.gradle

```diff
@@ -76,23 +76,18 @@ dependencies {
 
```

Comment:
Maybe it is better to create several logic groups of dependencies with some comments (if you really want to change smth here at all)